### PR TITLE
scrubbed files for correct use of acronyms

### PR DESCRIPTION
**Pull request by shortpatti**
Scrubbed all markdown files for proper use of acronyms (spell out first instance).

Fixed typos along the way.

### CODE REVIEW

File: backends/opentsdb/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/backends/opentsd
 
 # OpenTSDB with HTTP
 
-Netdata can easily communicate with OpenTSDB using HTTP API. To enable this channel, set the following options in your
+Netdata can easily communicate with Open Time Series Database (OpenTSDB) using HTTP API. To enable this channel, set the following options in your
```
Comment (joelhans) [inline backends/opentsdb/README.md]: I think the formal name is actually **OpenTSDB**, so we don't want to change that. We could, however, expand just **TSDB**.

```suggestion
Netdata can easily communicate with OpenTSDB, a time series database (TSDB) implementation, using HTTP API. To enable this channel, set the following options in your
```

File: build_external/README.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ For configurations that involve building and running the agent alone, we still u
 `docker-compose` for consistency with more complex configurations. The more complex
 configurations allow the agent to be run in conjunction with parts of the cloud
 infrastructure (these parts of the code are not public), or with external brokers
-(such as VerneMQ for MQTT), or with other external tools (such as TSDB to allow the agent to
+(such as VerneMQ for MQ Telemetry Transport (MQTT)), or with other external tools (such as Time Series Database (TSDB) to allow the agent to
```
Comment (joelhans) [inline build_external/README.md]: **Time series database** isn't a formal name, and so should be written in lowercase.

```suggestion
(such as VerneMQ for MQ Telemetry Transport (MQTT)), or with other external tools (such as a time series database (TSDB) to allow the agent to
```

File: collectors/perf.plugin/README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/perf.
 
 # perf.plugin
 
-`perf.plugin` collects system-wide CPU performance statistics from Performance Monitoring Units (PMU) using
+`perf.plugin` collects system-wide CPU performance statistics from Performance Monitoring Units (PMUs) using
```
Comment (joelhans) [inline collectors/perf.plugin/README.md]: I think this should be lowercase.

```suggestion
`perf.plugin` collects system-wide CPU performance statistics from performance monitoring units (PMUs) using
```

File: collectors/perf.plugin/README.md [hunk 4]
```diff
@@ -42,9 +42,9 @@ Hardware cache events:
 2.  L1D prefetch cache operations
 3.  L1I cache operations
 4.  LL cache operations
-5.  DTLB cache operations
-6.  ITLB cache operations
-7.  PBU cache operations
+5.  Data Translation Lookaside Buffer (DTLB) cache operations
+6.  Instruction Translation Lookaside Buffer (ITLB) cache operations
+7.  Proxy Binding Unit (PBU) cache operations
```
Comment (joelhans) [inline collectors/perf.plugin/README.md]: Lowercase these as well.

```suggestion
5.  Data translation lookaside Buffer (DTLB) cache operations
6.  Instruction translation lookaside Buffer (ITLB) cache operations
7.  Proxy binding unit (PBU) cache operations
```

File: collectors/python.d.plugin/boinc/README.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 
 # BOINC monitoring with Netdata
 
-Monitors task counts for the Berkely Open Infrastructure Networking Computing (BOINC) distributed computing client using the same RPC interface that the BOINC monitoring GUI does.
+Monitors task counts for the Berkely Open Infrastructure Networking Computing (BOINC) distributed computing client using the same Remote Procedure Call (RPC) interface that the BOINC monitoring GUI does.
```
Comment (joelhans) [inline collectors/python.d.plugin/boinc/README.md]:
```suggestion
Monitors task counts for the Berkely Open Infrastructure Networking Computing (BOINC) distributed computing client using the same remote procedure call (RPC) interface that the BOINC monitoring GUI does.
```

File: collectors/python.d.plugin/exim/README.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 
 # Exim monitoring with Netdata
 
-Simple module executing `exim -bpc` to grab exim queue.
+Simple module executing `exim -bpc` to grab the Experimental Internet Mailer (Exim) queue.
```
Comment (joelhans) [inline collectors/python.d.plugin/exim/README.md]: I think we should just leave this as **Exim**. Seems like that's the real name now.

File: collectors/python.d.plugin/nvidia_smi/README.md [hunk 7]
```diff
@@ -7,12 +7,12 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 
 # NVIDIA GPU monitoring with Netdata
 
-Monitors performance metrics (memory usage, fan speed, pcie bandwidth utilization, temperature, etc.) using `nvidia-smi` cli tool.
+Monitors performance metrics (memory usage, fan speed, Peripheral Component Interconnect Express (PCIe) bandwidth utilization, temperature, etc.) using `nvidia-smi` CLI tool.
```
Comment (joelhans) [inline collectors/python.d.plugin/nvidia_smi/README.md]: Maybe leave this as **PCI Express**? I feel like it's getting too wordy now.

```suggestion
Monitors performance metrics (memory usage, fan speed, PCI Express (PCIe) bandwidth utilization, temperature, etc.) using `nvidia-smi` CLI tool.
```

File: collectors/python.d.plugin/nvidia_smi/README.md [hunk 8]
```diff
@@ -7,12 +7,12 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
...
 
 
 **Requirements and Notes:**
 
--   You must have the `nvidia-smi` tool installed and your NVIDIA GPU(s) must support the tool. Mostly the newer high end models used for AI / ML and Crypto or Pro range, read more about [nvidia_smi](https://developer.nvidia.com/nvidia-system-management-interface).
+-   You must have the `nvidia-smi` tool installed and your NVIDIA GPU(s) must support the tool. Mostly the newer high end models used for Artificial Intelligence/Machine Learning (AI/ML) and Crypto or Pro range, read more about [nvidia_smi](https://developer.nvidia.com/nvidia-system-management-interface).
```
Comment (joelhans) [inline collectors/python.d.plugin/nvidia_smi/README.md]:
```suggestion
-   You must have the `nvidia-smi` tool installed and your NVIDIA GPU(s) must support the tool. Mostly the newer high end models used for artificial intelligence/machine learning (AI/ML) and Crypto or Pro range, read more about [nvidia_smi](https://developer.nvidia.com/nvidia-system-management-interface).
```

File: collectors/python.d.plugin/postfix/README.md [hunk 9]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 
 # Postfix monitoring with Netdata
 
-Monitors MTA email queue statistics using postqueue tool.  
+Monitors Mail Transfer Agent (MTA) email queue statistics using postqueue tool.
```
Comment (joelhans) [inline collectors/python.d.plugin/postfix/README.md]:
```suggestion
Monitors mail transfer agent (MTA) email queue statistics using postqueue tool.  
```

File: collectors/python.d.plugin/web_log/README.md [hunk 10]
```diff
@@ -13,7 +13,7 @@ Tails access log file and Collects web server/caching proxy metrics.
 
 Web server log files exist for more than 20 years. All web servers of all kinds, from all vendors, [since the time NCSA httpd was powering the web](https://en.wikipedia.org/wiki/NCSA_HTTPd), produce log files, saving in real-time all accesses to web sites and APIs.
 
-Yet, after the appearance of google analytics and similar services, and the recent rise of APM (Application Performance Monitoring) with sophisticated time-series databases that collect and analyze metrics at the application level, all these web server log files are mostly just filling our disks, rotated every night without any use whatsoever.
+Yet, after the appearance of google analytics and similar services, and the recent rise of Application Performance Monitoring (APM) with sophisticated time-series databases that collect and analyze metrics at the application level, all these web server log files are mostly just filling our disks, rotated every night without any use whatsoever.
```
Comment (joelhans) [inline collectors/python.d.plugin/web_log/README.md]:
```suggestion
Yet, after the appearance of google analytics and similar services, and the recent rise of application performance monitoring (APM) with sophisticated time-series databases that collect and analyze metrics at the application level, all these web server log files are mostly just filling our disks, rotated every night without any use whatsoever.
```

File: docs/Performance.md [hunk 11]
```diff
@@ -114,28 +114,28 @@ Edit `/etc/netdata/netdata.conf`, find the `[plugins]` section:
...
+    check for new plugins every = 60
 ```
 
 In detail:
 
 | plugin|description|
 |:----:|:----------|
 | `proc`|the internal plugin used to monitor the system. Normally, you don't want to disable this. You can disable individual functions of it at the next section.|
-| `tc`|monitoring network interfaces QoS (tc classes)|
+| `tc`|monitoring network interfaces Quality of Service (QoS) (tc classes)|
```
Comment (joelhans) [inline docs/Performance.md]:
```suggestion
| `tc`|monitoring network interfaces quality of service (QoS) (tc classes)|
```

File: docs/Running-behind-haproxy.md [hunk 12]
```diff
@@ -7,8 +7,8 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/Running-beh
 
 # Netdata via HAProxy
 
-> HAProxy is a free, very fast and reliable solution offering high availability, load balancing, 
-> and proxying for TCP and HTTP-based applications. It is particularly suited for very high traffic web sites 
+> High Availability Proxy (HAProxy) is a free, very fast and reliable solution offering high availability, load balancing,
```
Comment (joelhans) [inline docs/Running-behind-haproxy.md]: Again, given that HAProxy seems to be the official name, we don't want to mess with it.

```suggestion
> HAProxy (high-availability proxy) is a free, very fast and reliable solution offering high availability, load balancing, 
```

File: docs/Running-behind-haproxy.md [hunk 13]
```diff
@@ -64,7 +64,7 @@ A example where the base URL is used with a subpath `/netdata/`:
 
 ### Frontend
 
-To use a subpath, create an ACL, which will set a variable based on the subpath.
+To use a subpath, create an Access Control List (ACL), which sets a variable based on the subpath.
```
Comment (joelhans) [inline docs/Running-behind-haproxy.md]:
```suggestion
To use a subpath, create an access control list (ACL), which sets a variable based on the subpath.
```

File: docs/netdata-security.md [hunk 14]
```diff
@@ -56,9 +56,9 @@ Netdata is a monitoring system. It should be protected, the same way you protect
...
+This information is not sensitive (meaning that it is not your business data), but **it is important for possible attackers**. It will give them clues on what to check, what to try and in the case of Distributed Denial of Service (DDoS) against your applications, they will know if they are doing it right or not.
 
-Also, viewers could use Netdata itself to stress your servers. Although the Netdata daemon runs unprivileged, with the minimum process priority (scheduling priority `idle` - lower than nice 19) and adjusts its OutOfMemory (OOM) score to 1000 (so that it will be first to be killed by the kernel if the system starves for memory), some pressure can be applied on your systems if someone attempts a DDoS against Netdata.
+Also, viewers could use Netdata itself to stress your servers. Although the Netdata daemon runs unprivileged, with the minimum process priority (scheduling priority `idle` - lower than nice 19) and adjusts its Out-Of-Memory (OOM) score to 1000 (so that it will be first to be killed by the kernel if the system starves for memory), some pressure can be applied on your systems if someone attempts a DDoS against Netdata.
```
Comment (joelhans) [inline docs/netdata-security.md]:
```suggestion
Also, viewers could use Netdata itself to stress your servers. Although the Netdata daemon runs unprivileged, with the minimum process priority (scheduling priority `idle` - lower than nice 19) and adjusts its out of memory (OOM) score to 1000 (so that it will be first to be killed by the kernel if the system starves for memory), some pressure can be applied on your systems if someone attempts a DDoS against Netdata.
```

File: docs/netdata-security.md [hunk 15]
```diff
@@ -56,9 +56,9 @@ Netdata is a monitoring system. It should be protected, the same way you protect
...
 
-This information is not sensitive (meaning that it is not your business data), but **it is important for possible attackers**. It will give them clues on what to check, what to try and in the case of DDoS against your applications, they will know if they are doing it right or not.
+This information is not sensitive (meaning that it is not your business data), but **it is important for possible attackers**. It will give them clues on what to check, what to try and in the case of Distributed Denial of Service (DDoS) against your applications, they will know if they are doing it right or not.
```
Comment (joelhans) [inline docs/netdata-security.md]:
```suggestion
This information is not sensitive (meaning that it is not your business data), but **it is important for possible attackers**. It will give them clues on what to check, what to try and in the case of distributed denial of service (DDoS) against your applications, they will know if they are doing it right or not.
```

File: docs/netdata-security.md [hunk 16]
```diff
@@ -68,11 +68,11 @@ Until we add a distributed authentication method to Netdata, you have the follow
 
 #### Expose Netdata only in a private LAN
 
-If your organisation has a private administration and management LAN, you can bind Netdata on this network interface on all your servers. This is done in `Netdata.conf` with these settings:
+If your organisation has a private administration and management Local Area Network (LAN), you can bind Netdata on this network interface on all your servers. This is done in `Netdata.conf` with these settings:
```
Comment (joelhans) [inline docs/netdata-security.md]:
```suggestion
If your organization has a private administration and management local area network (LAN), you can bind Netdata on this network interface on all your servers. This is done in `netdata.conf` with these settings:
```

File: docs/tutorials/collect-apache-nginx-web-logs.md [hunk 17]
```diff
@@ -12,14 +12,14 @@ performance of their web servers, and Netdata is taking important steps to make
 
 By parsing web server log files with Netdata, and seeing the volume of redirects, requests, or server errors over time,
 you can better understand what's happening on your infrastructure. Too many bad requests? Maybe a recent deploy missed a
-few small SVG icons. Too many requsests? Time to batten down the hatches—it's a DDoS.
+few small SVG icons. Too many requsests? Time to batten down the hatches—it's a Distributed Denial of Service (DDoS).
```
Comment (joelhans) [inline docs/tutorials/collect-apache-nginx-web-logs.md]:
```suggestion
few small SVG icons. Too many requsests? Time to batten down the hatches—it's a distributed denial of service (DDoS).
```

File: docs/tutorials/collect-apache-nginx-web-logs.md [hunk 18]
```diff
@@ -12,14 +12,14 @@ performance of their web servers, and Netdata is taking important steps to make
...
 
 Netdata has been capable of monitoring web log files for quite some time, thanks for the [weblog python.d
 module](../../collectors/python.d.plugin/web_log/README.md), but we recently refactored this module in Go, and that
 effort comes with a ton of improvements.
 
-You can now use the [LTSV log format](http://ltsv.org/), track TLS and cipher usage, and the whole parser is faster than
-ever. In one test on a system with SSD storage, the collector consistently parsed the logs for 200,000 requests in
+You can now use the [LTSV log format](http://ltsv.org/), track Transport Layer Security (TLS) and cipher usage, and the whole parser is faster than
+ever. In one test on a system with Solid State Drive (SSD) storage, the collector consistently parsed the logs for 200,000 requests in
```
Comment (joelhans) [inline docs/tutorials/collect-apache-nginx-web-logs.md]:
```suggestion
ever. In one test on a system with solid state drive (SSD) storage, the collector consistently parsed the logs for 200,000 requests in
```

File: docs/why-netdata/1s-granularity.md [hunk 19]
```diff
@@ -11,7 +11,7 @@ High resolution metrics are required to effectively monitor and troubleshoot sys
 
 ## Why?
 
--   The world is going real-time. Today, customer experience is significantly affected by response time, so SLAs are tighter than ever before. It is just not practical to monitor a 2-second SLA with 10-second metrics.
+-   The world is going real-time. Today, customer experience is significantly affected by response time, so Service Level Agreements (SLAs) are tighter than ever before. It is just not practical to monitor a 2-second SLA with 10-second metrics.
```
Comment (joelhans) [inline docs/why-netdata/1s-granularity.md]:
```suggestion
-   The world is going real-time. Today, customer experience is significantly affected by response time, so service-level agreements (SLAs) are tighter than ever before. It is just not practical to monitor a 2-second SLA with 10-second metrics.
```

File: docs/why-netdata/README.md [hunk 20]
```diff
@@ -15,7 +15,7 @@ Netdata is built around 4 principles:
 
 1.  **[Per second data collection for all metrics.](1s-granularity.md)**
 
-    _It is impossible to monitor a 2 second SLA, with 10 second metrics._
+    _It is impossible to monitor a 2 second Service Level Agreement (SLA), with 10 second metrics._
```
Comment (joelhans) [inline docs/why-netdata/README.md]:
```suggestion
    _It is impossible to monitor a 2 second service-level agreement (SLA), with 10-second metrics._
```

File: health/notifications/email/README.md [hunk 21]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/health/notificat
 
 # Email
 
-You need a working `sendmail` command for email alerts to work. Almost all MTAs provide a `sendmail` interface.
+You need a working `sendmail` command for email alerts to work. Almost all Mail Transfer Agents (MTAs) provide a `sendmail` interface.
```
Comment (joelhans) [inline health/notifications/email/README.md]:
```suggestion
You need a working `sendmail` command for email alerts to work. Almost all mail transfer agents (MTAs) provide a `sendmail` interface.
```

File: collectors/python.d.plugin/am2320/README.md [hunk 22]
```diff
@@ -41,7 +41,7 @@ Raspberry Pi 3B/4 Pins:
 - Board GND (pin 6) to sensor GND (pin 3)
 - Board SCL (pin 5) to sensor SCL (pin 4)
 
-You may also need to add two I2C pullup resistors if your board does not already have them. The Raspberry Pi does have internal pullup resistors but it doesn't hurt to add them anyway. You can use 2.2K - 10K but we will just use 10K. The resistors go from VDD to SCL and SDA each.
+You may also need to add two I2C pullup resistors if your board does not already have them. The Raspberry Pi does have internal pullup resistors but it doesn't hurt to add them anyway. You can use 2.2K - 10K but we will just use 10K. The resistors go from Voltage Drain Drain (VDD) to Serial Clock (SCL) and Serial Data (SDA) each.
```
Comment (shortpatti) [inline collectors/python.d.plugin/am2320/README.md]:
```suggestion
You may also need to add two I2C pullup resistors if your board does not already have them. The Raspberry Pi does have internal pullup resistors but it doesn't hurt to add them anyway. You can use 2.2K - 10K but we will just use 10K. The resistors go from Voltage Drain Drain (VDD) to Serial Clock (SCL) and Serial Data Access (SDA) each.
```

File: collectors/perf.plugin/README.md [hunk 23]
```diff
@@ -42,9 +42,9 @@ Hardware cache events:
 2.  L1D prefetch cache operations
 3.  L1I cache operations
 4.  LL cache operations
-5.  DTLB cache operations
-6.  ITLB cache operations
-7.  PBU cache operations
+5.  Data translation lookaside Buffer (DTLB) cache operations
```
Comment (shortpatti) [inline collectors/perf.plugin/README.md]:
```suggestion
5.  DTLB cache operations
```

File: collectors/perf.plugin/README.md [hunk 24]
```diff
@@ -43,8 +43,8 @@ Hardware cache events:
 3.  L1I cache operations
 4.  LL cache operations
 5.  DTLB cache operations
-6.  ITLB cache operations
-7.  PBU cache operations
+6.  Instruction translation lookaside Buffer (ITLB) cache operations
```
Comment (shortpatti) [inline collectors/perf.plugin/README.md]:
```suggestion
6.  ITLB cache operations
```

File: collectors/perf.plugin/README.md [hunk 25]
```diff
@@ -44,7 +44,7 @@ Hardware cache events:
 4.  LL cache operations
 5.  DTLB cache operations
 6.  ITLB cache operations
-7.  PBU cache operations
+7.  Proxy binding unit (PBU) cache operations
```
Comment (shortpatti) [inline collectors/perf.plugin/README.md]:
```suggestion
7.  PBU cache operations
```

File: collectors/python.d.plugin/boinc/README.md [hunk 26]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 
 # BOINC monitoring with Netdata
 
-Monitors task counts for the Berkely Open Infrastructure Networking Computing (BOINC) distributed computing client using the same RPC interface that the BOINC monitoring GUI does.
+Monitors task counts for the Berkely Open Infrastructure Networking Computing (BOINC) distributed computing client using the same remote procedure call (RPC) interface that the BOINC monitoring GUI does.
```
Comment (shortpatti) [inline collectors/python.d.plugin/boinc/README.md]:
```suggestion
Monitors task counts for the Berkely Open Infrastructure Networking Computing (BOINC) distributed computing client using the same Remote Procedure Call (RPC) interface that the BOINC monitoring GUI does.
```

File: collectors/python.d.plugin/fail2ban/README.md [hunk 27]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (shortpatti) [inline collectors/python.d.plugin/fail2ban/README.md]:
```suggestion

```

File: collectors/python.d.plugin/exim/README.md (hunk 6 above)
Comment (shortpatti) [inline collectors/python.d.plugin/exim/README.md]:
```suggestion
Simple module executing `exim -bpc` to grab the Exim queue.
```

File: collectors/python.d.plugin/bind_rndc/README.md [hunk 28]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (shortpatti) [inline collectors/python.d.plugin/bind_rndc/README.md]:
```suggestion

```

File: collectors/python.d.plugin/bind_rndc/README.md [hunk 29]
```diff
@@ -6,12 +7,15 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 -->
 
 # ISC Bind monitoring with Netdata
+=======
```
Comment (shortpatti) [inline collectors/python.d.plugin/bind_rndc/README.md]:
```suggestion

```

File: collectors/python.d.plugin/bind_rndc/README.md [hunk 30]
```diff
@@ -6,12 +7,15 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
...
+# ISC BIND monitoring with Netdata
```
Comment (shortpatti) [inline collectors/python.d.plugin/bind_rndc/README.md]:
```suggestion

```

File: collectors/python.d.plugin/bind_rndc/README.md [hunk 31]
```diff
@@ -6,12 +7,15 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
...
+>>>>>>> scrubbing the collectors for correct use of acronyms
```
Comment (shortpatti) [inline collectors/python.d.plugin/bind_rndc/README.md]:
```suggestion

```

File: collectors/python.d.plugin/fail2ban/README.md [hunk 32]
```diff
@@ -6,8 +6,11 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 -->
 
 # Fail2ban monitoring with Netdata
+=======
```
Comment (shortpatti) [inline collectors/python.d.plugin/fail2ban/README.md]:
```suggestion

```

File: collectors/python.d.plugin/fail2ban/README.md [hunk 33]
```diff
@@ -6,8 +6,11 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
...
+# Fail2Ban monitoring with Netdata
```
Comment (shortpatti) [inline collectors/python.d.plugin/fail2ban/README.md]:
```suggestion

```

File: collectors/python.d.plugin/fail2ban/README.md [hunk 34]
```diff
@@ -6,8 +6,11 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
...
+>>>>>>> scrubbing the collectors for correct use of acronyms
```
Comment (shortpatti) [inline collectors/python.d.plugin/fail2ban/README.md]:
```suggestion

```

File: collectors/python.d.plugin/riakkv/README.md [hunk 35]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (shortpatti) [inline collectors/python.d.plugin/riakkv/README.md]:
```suggestion

```

File: collectors/python.d.plugin/riakkv/README.md [hunk 36]
```diff
@@ -6,6 +7,9 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 -->
 
 # RiakKV monitoring with Netdata
+=======
```
Comment (shortpatti) [inline collectors/python.d.plugin/riakkv/README.md]:
```suggestion

```

File: collectors/python.d.plugin/riakkv/README.md [hunk 37]
```diff
@@ -6,6 +7,9 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
...
+# Riak KV monitoring with Netdata
```
Comment (shortpatti) [inline collectors/python.d.plugin/riakkv/README.md]:
```suggestion

```

File: collectors/python.d.plugin/riakkv/README.md [hunk 38]
```diff
@@ -6,6 +7,9 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
...
+>>>>>>> scrubbing the collectors for correct use of acronyms
```
Comment (shortpatti) [inline collectors/python.d.plugin/riakkv/README.md]:
```suggestion

```

File: docs/Running-behind-apache.md [hunk 39]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (shortpatti) [inline docs/Running-behind-apache.md]:
```suggestion

```

File: docs/Running-behind-apache.md [hunk 40]
```diff
@@ -6,8 +7,11 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/Running-beh
 -->
 
 # Netdata via apache's mod_proxy
+=======
```
Comment (shortpatti) [inline docs/Running-behind-apache.md]:
```suggestion

```

File: docs/Running-behind-apache.md [hunk 41]
```diff
@@ -6,8 +7,11 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/Running-beh
...
+# Netdata via Apache's mod_proxy
```
Comment (shortpatti) [inline docs/Running-behind-apache.md]:
```suggestion

```

File: docs/Running-behind-apache.md [hunk 42]
```diff
@@ -6,8 +7,11 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/Running-beh
...
+>>>>>>> scrubbing the collectors for correct use of acronyms
```
Comment (shortpatti) [inline docs/Running-behind-apache.md]:
```suggestion

```

File: docs/netdata-security.md (hunk 16 above)
Comment (shortpatti) [inline docs/netdata-security.md]:
```suggestion
If your organisation has a private administration and management LAN, you can bind Netdata on this network interface on all your servers. This is done in `Netdata.conf` with these settings:
```

File: docs/tutorials/collect-apache-nginx-web-logs.md [hunk 43]
```diff
@@ -12,14 +12,14 @@ performance of their web servers, and Netdata is taking important steps to make
...
+few small SVG icons. Too many requsests? Time to batten down the hatches—it's a distributed denial of service (DDoS).
 
 Netdata has been capable of monitoring web log files for quite some time, thanks for the [weblog python.d
 module](../../collectors/python.d.plugin/web_log/README.md), but we recently refactored this module in Go, and that
 effort comes with a ton of improvements.
 
-You can now use the [LTSV log format](http://ltsv.org/), track TLS and cipher usage, and the whole parser is faster than
-ever. In one test on a system with SSD storage, the collector consistently parsed the logs for 200,000 requests in
+You can now use the [LTSV log format](http://ltsv.org/), track Transport Layer Security (TLS) and cipher usage, and the whole parser is faster than
+ever. In one test on a system with solid state drive (SSD) storage, the collector consistently parsed the logs for 200,000 requests in
```
Comment (shortpatti) [inline docs/tutorials/collect-apache-nginx-web-logs.md]:
```suggestion
ever. In one test on a system with SSD storage, the collector consistently parsed the logs for 200,000 requests in
```

File: docs/tutorials/collect-apache-nginx-web-logs.md [hunk 44]
```diff
@@ -12,14 +12,14 @@ performance of their web servers, and Netdata is taking important steps to make
 
 By parsing web server log files with Netdata, and seeing the volume of redirects, requests, or server errors over time,
 you can better understand what's happening on your infrastructure. Too many bad requests? Maybe a recent deploy missed a
-few small SVG icons. Too many requsests? Time to batten down the hatches—it's a DDoS.
+few small SVG icons. Too many requsests? Time to batten down the hatches—it's a distributed denial of service (DDoS).
```
Comment (shortpatti) [inline docs/tutorials/collect-apache-nginx-web-logs.md]:
```suggestion
few small SVG icons. Too many requsests? Time to batten down the hatches—it's a Distributed Denial of Service (DDoS).
```

File: docs/why-netdata/1s-granularity.md [hunk 45]
```diff
@@ -11,7 +11,7 @@ High resolution metrics are required to effectively monitor and troubleshoot sys
 
 ## Why?
 
--   The world is going real-time. Today, customer experience is significantly affected by response time, so SLAs are tighter than ever before. It is just not practical to monitor a 2-second SLA with 10-second metrics.
+-   The world is going real-time. Today, customer experience is significantly affected by response time, so service-level agreements (SLAs) are tighter than ever before. It is just not practical to monitor a 2-second SLA with 10-second metrics.
```
Comment (shortpatti) [inline docs/why-netdata/1s-granularity.md]:
```suggestion
-   The world is going real-time. Today, customer experience is significantly affected by response time, so SLAs are tighter than ever before. It is just not practical to monitor a 2-second SLA with 10-second metrics.
```

File: docs/why-netdata/README.md [hunk 46]
```diff
@@ -15,7 +15,7 @@ Netdata is built around 4 principles:
 
 1.  **[Per second data collection for all metrics.](1s-granularity.md)**
 
-    _It is impossible to monitor a 2 second SLA, with 10 second metrics._
+    _It is impossible to monitor a 2 second service-level agreement (SLA), with 10-second metrics._
```
Comment (shortpatti) [inline docs/why-netdata/README.md]:
```suggestion
    _It is impossible to monitor a 2 second SLA, with 10-second metrics._
```

File: health/notifications/email/README.md [hunk 47]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/health/notificat
 
 # Email
 
-You need a working `sendmail` command for email alerts to work. Almost all MTAs provide a `sendmail` interface.
+You need a working `sendmail` command for email alerts to work. Almost all mail transfer agents (MTAs) provide a `sendmail` interface.
```
Comment (shortpatti) [inline health/notifications/email/README.md]:
```suggestion
You need a working `sendmail` command for email alerts to work. Almost all Mail Transfer Agents (MTAs) provide a `sendmail` interface.
```

File: collectors/python.d.plugin/sensors/README.md [hunk 48]
```diff
@@ -23,7 +23,7 @@ sudo ./edit-config python.d/sensors.conf
 
 ### possible issues
```
Comment (shortpatti) [inline collectors/python.d.plugin/sensors/README.md]:
```suggestion
### Possible issues
```

File: collectors/perf.plugin/README.md [hunk 49]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/perf.
 
 # perf.plugin
 
-`perf.plugin` collects system-wide CPU performance statistics from Performance Monitoring Units (PMU) using
+`perf.plugin` collects system-wide CPU performance statistics from performance monitoring units (PMUs) using
```
Comment (shortpatti) [inline collectors/perf.plugin/README.md]:
```suggestion
`perf.plugin` collects system-wide CPU performance statistics from Performance Monitoring Units (PMUs) using
```

File: collectors/python.d.plugin/chrony/README.md [hunk 50]
```diff
@@ -7,7 +7,7 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/pytho
 
 # Chrony monitoring with Netdata
 
-Monitors the precision and statistics of a local chronyd server, and produces:
+Monitors the precision and statistics of a local Chrony Daemon (chronyd) server, and produces:
```
Comment (shortpatti) [inline collectors/python.d.plugin/chrony/README.md]:
```suggestion
Monitors the precision and statistics of a local chronyd server, and produces:
```
